bindings/python/unicorn/unicorn.py

```diff
@@ -17,12 +17,12 @@
 if _python2:
     range = xrange
 
-_lib_path = os.path.split(__file__)[0]
-_all_libs = (
-    "unicorn.dll",
-    "libunicorn.so",
-    "libunicorn.dylib",
-)
+if sys.platform == 'darwin':
+    _lib = "libunicorn.1.dylib"
```

Comment:
the part `.1` depends on the Unicorn version. for example, when we bump version to 2.0, the lib will be `libunicorn.2.dylib`. how about using libunicorn.dylib to avoid hardcode the version here (this is the current approach)?